File: eRents.UI/e_rents_mobile/lib/models/review_notification_message.dart
class ReviewNotificationMessage {
  final int propertyId;
  final int reviewId;
  final String? message;

  ReviewNotificationMessage({required this.propertyId, required this.reviewId, this.message});

  factory ReviewNotificationMessage.fromJson(Map<String, dynamic> json) {
    return ReviewNotificationMessage(
      propertyId: json['propertyId'],
      reviewId: json['reviewId'],
      message: json['message'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'propertyId': propertyId,
      'reviewId': reviewId,
      'message': message,
    };
  }
}

File: eRents.Shared/DTOs/ReviewNotificationMessage.cs
namespace eRents.Shared.DTOs;

/// <summary>
/// Review notification message DTO
/// Used for review-related notifications across services
/// </summary>
public class ReviewNotificationMessage
{
    public int? ReviewId { get; set; }
    public string? Message { get; set; }
    public int? PropertyId { get; set; }
    public string? UserId { get; set; }
    public decimal? Rating { get; set; }
    public string? ReviewText { get; set; }
} 